default/scripting/species/common/telepathic.macros

```diff
@@ -0,0 +1,15 @@
+TELEPATHIC_DETECTION
+'''     EffectsGroup
+            scope = And [
+                PopulationCenter
+                Not OR [
+                    Species name = Source.Species
+                    Species name = "SP_EXOBOT"
+                ]
+                Not Population high = 0
+                WithinStarlaneJumps jumps = @1@ condition = Source
```

Comment:
Um, that's a macro parameter. They don't get explicitely defined, you just use them. When the macro is called, you need to pass a parameter, which will replace the "@1@" here. See my comment above below the line where this macro is called for the Trith species.